### ISSUE TITLE
formalise processing of landsat 7 data

### ISSUE THREAD

**Issue by niallmcc** [surftemp/landsat_importer#3]
I am trying to incorporate landsat7 data into workflows to blend data from landsat7/8/9, as required for the greenland lakes project.

Currently landsat_importer is mapping landsat7 bands to the closest landsat8/9 equivalents.  This is most confusing when considering the L7 bands B6_VCID_1 and B6_VCID_2 and mapping them to L8/9 bands 10 and 11.

From the initial implementation last month, I think landsat_importer is probably not the best place to be doing this.  Another workflow component should be managing this mapping.  The usgs_group stage already has the ability to rename variables as well as combining L1/L2 scenes.

**Comment by oembury** [comment]
Agreed - the landsat_importer should not attempt any band remapping / renaming to make L7 look like L8/9. Landsat 7 carries a different instrument (ETM+) than Landsats 8 & 9 (OLI and TIRS), so gbcs etc would need to process it as "landsat 7" data.

The importer could assign appropriate attributes to make later mapping / reading of "common" bands easier e.g. `band_name` or `long_name` could be set ("blue" for band 1 on landsat 7, and band 2 on 8&9)

Note - earlier landsats only have a single thermal channel:
![Landsat bands](https://landsat.usgs.gov/sites/default/files/images/MSS-thru-L9-wavelengths.jpg)

However, landsat 7 does have high and low gain versions of band 6 thermal. It maybe appropriate to combine these in to a single band 6 if the importer is performing the conversion to brightness temperature.

**Comment by niallmcc** [comment]
Yes I think combining B6_VCID_1 and B6_VCID_2 into a B6 would be reasonable (probably a good idea to expose B6_VCID_1 and B6_VCID_2 separately if requested)

[https://www.usgs.gov/faqs/why-do-landsat-7-level-1-products-contain-two-thermal-bands](https://www.usgs.gov/faqs/why-do-landsat-7-level-1-products-contain-two-thermal-bands)

I can imagine the following naive way at least:

B6_VCID_2_threshold_min = minimum value that B6_VCID_2 can sense
B6_VCID_2_threshold_max = maximum value that B6_VCID_2 can sense

B6 = if (B6_VCID_2 >= B6_VCID_2_threshold_min and B6_VCID_2 <= B6_VCID_2_threshold_max) 
        then B6_VCID_2 
        else B6_VCID_1
